client/assets/js/admin-controller.js

```diff
@@ -74,44 +72,44 @@ angular.module('portal').controller('adminController', function ($route, $scope,
     }
 
     $scope.adminUpdate = function () {
-        $scope.projectsUpdated = [];
-        $scope.internalError = false;
+        let projectsUpdated = [];
+        let internalError = false;
 
         $scope.projects.forEach(function(project) {
 
-            $scope.projectMatches = 0;
+            let projectMatches = 0;
             $scope.projectsOldState.forEach(function(projectOld) {
                 if (project._id === projectOld._id) {
-                    $scope.projectMatches += 1;
+                    projectMatches += 1;
                     // if (project.private.status !== projectOld.private.status) {
```

Comment:
could you remove these comments for code cleanliness? 😄

Reply:
I want to leave this one in because it would be cool to implement this later but i can remove it if you want

Reply:
ah makes sense, didn't realize it was a future feature